php-translation/symfony-bundle/0.4/config/packages/php_translation.yaml

```diff
@@ -0,0 +1,14 @@
+translation:
+    locales: ["en", "sv", "fr"]
+    webui:
+        enabled: true
+    edit_in_place:
+        enabled: false
+        config_name: app
+        activator: php_translation.edit_in_place.activator
```

Comment:
Could we remove this `activator` option and rely on a default value? It's super redundant that the value of the `activator` option for `edit_in_place` option of PGP translation is `php_translation.edit_in_place.activator`.